lib/fastlane/plugin/wpmreleasetoolkit/actions/common/create_release_backmerge_pull_request_action.rb

```diff
@@ -0,0 +1,163 @@
+        end
+
+        github_helper = Fastlane::Helper::GithubHelper.new(github_token: params[:github_token])
+        target_milestone = milestone_title.nil? ? nil : github_helper.get_milestone(repository, milestone_title)
+
+        target_branches = if target_branches_param.empty?
+                            determine_target_branches(release_branch.delete('release/'), default_branch)
+                          else
+                            target_branches_param
+                          end
+
+        pr_urls = []
+
+        target_branches.each do |target_branch|
+          Fastlane::Helper::GitHelper.checkout_and_pull(release_branch)
+
+          pr_urls << create_backmerge_pr(
+            token: token,
+            repository: params[:repository],
+            title: "Merge #{release_branch} into #{target_branch}",
+            head_branch: release_branch,
+            base_branch: target_branch,
+            milestone: target_milestone&.number,
+            labels: labels
+          )
+        end
+
+        pr_urls
+      end
+
+      def self.determine_target_branches(release_version, default_branch)
+        release_branches = Actions.sh('git branch -r -l "origin/release/*"').chomp
+
+        all_release_branches_versions = release_branches
+                                        .split("\n")
+                                        .map { |branch| branch.match(%r{origin/release/([0-9.]*)})&.captures&.first }
+                                        .compact
+
+        target_branches = all_release_branches_versions.select { |branch| Gem::Version.new(branch) > Gem::Version.new(release_version) }
+                                                       .map { |v| "release/#{v}" }
+        target_branches = [default_branch] if target_branches.empty?
+
+        target_branches
+      end
+
+      def self.create_backmerge_pr(token:, repository:, title:, head_branch:, base_branch:, milestone:, labels:)
+        intermediate_branch = "merge/#{head_branch.gsub('/', '-')}-into-#{base_branch.gsub('/', '-')}"
+        Fastlane::Helper::GitHelper.create_branch(intermediate_branch)
+
+        other_action.push_to_git_remote(tags: false)
+
+        pr_body = <<~BODY
+          Merging `#{head_branch}` into `#{base_branch}`.
+
+          Via intermediate branch `#{intermediate_branch}`, to help fix conflicts if any:
+          ```
+          #{head_branch.rjust(40)}  ----o-- - - -
+          #{' ' * 40}       \\
+          #{intermediate_branch.rjust(40)}        `---.
+          #{' ' * 40}             \\
+          #{base_branch.rjust(40)}  ------------x- - -
+          ```
+        BODY
+
+        other_action.create_pull_request(
+          api_token: token,
+          repo: repository,
+          title: title,
+          body: pr_body,
+          head: intermediate_branch,
+          base: base_branch,
+          labels: labels,
+          milestone: milestone
+        )
```

Comment:
> We need to protect against the case where there's nothing to merge.

I think even in cases of a cherry-pick, there's always at least a version bump?
The scenario where I see it happening is when we're talking about non-release branches, without a version bump. Still, that seems quite rare. Doesn't hurt to add a check though (as the call to the API will indeed fail) 🤔 

> Though maybe in those cases it's better for the action to fail explicitly

We have this case of the language fetch on Tumblr, where we force-push the new changes + try to create the PR, which I don't think even fails when a PR already exists (I think it just logs an error?).

Reply:
No there's also the case of a retry. e.g. `finalize_release` -> creates the backmerge PR, the RM merges that backmerge PR, but then the trampoline build fails for some reason (e.g. ASC or PSC API outage or similar unrelated to our side of the code or something that would need to be fixed in `buildkite-ci` or somewhere else but would not need any commit on the `release/*` branch of the repo to fix it…), and then the RM retries the `finalize_release`, which will try to create a new backmerge PR… but since the first one has already been created and merged and there hasn't been anything new pushed on `release/*` branch since, then PR would be empty and cause a GitHub API error in return.

Reply:
Added diff validation on 42b2ad44f83de2869e1b41360b21d40891931c55